### ISSUE TITLE
Cannot read handler of undefined

### ISSUE THREAD

**Issue by donguyenthanhlan** [JamesCullum/Safer-MyMonero#3]
When sending XMR out, there is an error as below. 
![image](https://user-images.githubusercontent.com/45186117/57894229-4c1ab600-7870-11e9-912e-da8becace229.png)

I take a look into the js code. Data is undefined
![image](https://user-images.githubusercontent.com/45186117/57894264-740a1980-7870-11e9-86bd-b25b996c92cd.png)

**Comment by JamesCullum** [comment]
The addon hasn't been updated for some time.
I assume that this is why it broke.
Doesn't the current MyMonero run as a software?

**Comment by donguyenthanhlan** [comment]
Yes, but I find your chrome extensions is much more convenient to use. Could you please help with the fix?

**Comment by JamesCullum** [comment]
When you say "help" you mean assist or do it?

**Comment by donguyenthanhlan** [comment]
I can do it. Could you guide please? I don't know what the issue is.

**Comment by JamesCullum** [comment]
Generally this is just a cached version of the web wallet.
You'd need to pull a new copy of all files locally and provide them in the plugin.
You can start by going to https://wallet.mymonero.com/ and trying to save it locally.
Whatever breaks needs to be fixed, so that it runs locally.

**Comment by donguyenthanhlan** [comment]
Got it. Thank man
